src/Microsoft.TemplateEngine.Edge/Settings/GlobalSettings.cs

```diff
@@ -1,52 +1,41 @@
-using Microsoft.TemplateEngine.Abstractions;
-using Microsoft.TemplateEngine.Abstractions.GlobalSettings;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
 using System;
 using System.Collections.Generic;
-using System.IO;
-using System.Text;
+using Microsoft.TemplateEngine.Abstractions.GlobalSettings;
+using Newtonsoft.Json;
 
 namespace Microsoft.TemplateEngine.Edge.Settings
 {
     public class GlobalSettings : IGlobalSettings
     {
         [JsonProperty]
-        public string DefaultLanguage { get; set; }
+        private List<TemplatesSourceData> _userInstalledTemplatesSources = new List<TemplatesSourceData>();
```

Comment:
Can you check settings.json content I think it will have `_userInstalledTemplatesSources`, use `JsonPropert("NiceName")`